_includes/footer.html

```diff
@@ -10,3 +10,17 @@
     </div>
   </div>
 </footer>
```

Comment:
the more I think about this footer image I think that some other jgs might feel personally more suited to the location and my interpretation of look and feel. I'll add to comment at end of review

Reply:
I went with the second jgs piece your comment.